loggers/README.md

```diff
@@ -4,48 +4,42 @@ A service to help log messages in the console or write logs to a file.
 
 ## Table of Contents
 
-- [ConsoleLogger](#consolelogger)
-- [FileLogger](#filelogger)
+- [Creating a Looger](#creating-a-logger)
```

Comment:
what's a Looger lmao

Reply:
Get with the times eric, it's the new name for it of course

Reply:
https://github.com/drashland/services/pull/21/commits/f932a16c646be2afebba9da556aeb1913c6b441e

Reply:
LOL